Documentation/subcommands/run-prepared.md

```diff
@@ -40,3 +40,23 @@ c9fad0e6    etcd    coreos.com/etcd prepared
 [25703.013405] etcd[4]: 2015/10/01 14:44:10 raft.node: ce2a822cea30bfca elected leader ce2a822cea30bfca at term 2
 [25703.017089] etcd[4]: 2015/10/01 14:44:10 etcdserver: published {Name:default ClientURLs:[http://localhost:2379 http://localhost:4001]} to cluster 7e27652122e8b2ae
 ```
+
+## Options
+
+| Flag | Default | Options | Description |
+| --- | --- | --- | --- |
+| `--interactive` |  `false` | `true` or `false` | Run pod interactively. If true, only one image may be supplied |
+| `--mds-register` |  `false` | `true` or `false` | Register pod with metadata service. It needs network connectivity to the host (`--net=(default|default-restricted|host)` |
+| `--net` |  `default` | A comma-separated list of networks. Syntax: `--net[=n[:args], ...]` | Configure the pod's networking and optionally pass a list of user-configured networks to oad and arguments to pass to them |
```

Comment:
Let's try this...

Configure the pod's networking. Optionally, pass a list of user-configured networks to load and a set arguments to pass to each network, respectively.